obj.go

```diff
@@ -297,6 +331,23 @@ func (t ObjectType) NewObject() (*Object, error) {
 	return &Object{ObjectType: t, dpiObject: obj}, nil
 }
 
+// Close releases a reference to the object type.
+func (t ObjectType) Close() error {
```

Comment:
If we change t, we need to use a pointer receiver (t *ObjectType)

Reply:
Ok